skcosmo/preprocessing/FlexibleScaler.py

```diff
@@ -0,0 +1,242 @@
+import numpy as np
+import sklearn as sk
+from sklearn.base import TransformerMixin, BaseEstimator
+
+
+class StandardFlexibleScaler(TransformerMixin, BaseEstimator):
+    """A more flexible scaler for features and data, that mimics
+    sklearn.preprocessing.StandardScaler make the mean of the
+    columns equal to zero and the variance of each column
+    (if designated) equal to one.
+
+    :param with_mean: If True, center the data before scaling
+    :type with_mean: boolean
+    :param with_std: If True, scale the data to unit variance
+    :type with_std: boolean
+    :param column_wise: If True, normalize each column separately
+    :type column_wise: boolean
+    """
+
+    def __init__(self, *, with_mean=True, with_std=True, column_wise=False):
```

Comment:
Is there a reason these are `keyword-only`?

Reply:
What do you mean?

Reply:
Using `*` in the`_init_` declaration means that you cannot call `StandardFlexibleScaler(True, True, True)`, you _have to_ call `StandardFlexibleScaler(with_mean=True, with_std=True, column_wise=True)`. I'm not against it, just curious.